packages/devcmd/src/process/ProcessExecutor.ts

```diff
@@ -51,6 +77,25 @@ export class ProcessExecutor {
   /**
    * Executes multiple processes in parallel and throws an exception if the exit code is non-zero.
    * Outputs (stdout/stderr) of the processes are sent to our stdout/stderr.
+   * 
+   * @param {{ [id: string]: ProcessInfo } | { [id: number]: ProcessInfo }} processMap A map linking process ids to {@link ProcessInfo} instances
```

Comment:
good thing we talk about this!
the type and name of this parameter might be slightly intransparent, but aside from a name-based map, you can also just pass an array of ProcessInfo here (the int-indexed object type is compatible with arrays).

example (feel free to add this to `process.test.ts`):
```ts
test("succeeding processes as array works", async () => {
  const execPipedParallel = new ProcessExecutor(nullConsole).execPipedParallel;
  await execPipedParallel([
    { command: "node", args: ["--version"] },
    { command: withCmdOnWin("npm"), args: ["--version"] },
  ]);
});
```

Therefore, a slightly more details description here might be worthwhile, e.g. something like "`The {@link ProcessInfo}s to execute. Can be a ProcessInfo array or a map-object. In both cases, the indices are used to identify the child processes in log outputs.`"

Reply:
Thats interesting, i didnt know that! 
I thought you would always have to provide ids yourself!